Add manual testing (#223)
AY2122S1-CS2103T-T11-3/tp: docs/DeveloperGuide.md

    1. While the app is still open, enter `exit` in the command box or click on the close window button.
        Expected: The application closes.

### Adding a patient <a name="add-patient-manual-testing"/>
1. Adding a doctor
    1. Prerequisites: `toggle` to the `Patients` tab. The following patient must already exists:
       John Doe; Phone: 98765432; Email: johnd@example.com; Address: 311, Clementi Ave 2, #02-25; Date of Birth: 20/7/1964; Tags: vaccinated; Risk: LOW
       
    2. Test case: `add n/Bob Doe hp/98765432 eml/johnd@example.com a/311, Clementi Ave 2, #02-25 dob/20/07/1964 t/vaccinated risk/LOW`
       Expected: Patient named Bob Doe is added successfully. Details of the added patient are shown in the status message.

    3. Test case: `add n/Bobby Doe hp/98765432 eml/johnd@example.com a/311, Clementi Ave 2, #02-25 dob/20/07/1964`

>         Expected: Patient named Bob Doe is added successfully. Details of the added patient are shown in the status message.

Reviewer: There is an inconsistency between Bobby and Bob here.

       Expected: No Patient is edited. Error message is shown in the response box: "This patient already exists in PlannerMD"

    7. Test case: `add n/John Doe hp/98765432 eml/johnd@example.com a/311, Kent Ridge Ave 2, #02-25 dob/25/12/0000 t/vaccinated risk/LOW`
       Expected: No Patient is edited. Error message is shown in the response box: "This patient already exists in PlannerMD"

    8. Other incorrect edit commands to try: `add`, `add n/Bob hp/`, `add n/` (not all fields, except tags and risk, are filled)<br>
       Expected: Similar to previous.

### Deleting a patient  <a name="delete-patient-manual-testing"/>
1. Deleting a patient while all doctors are being shown
    1. Prerequisites: `toggle` to the `Patients` tab. List all patients using the `list` command. There must be multiple patients. 

    2. Test case: `delete 1`<br>
       Expected: First patient is deleted from the patients list. Details of the deleted patient are shown in the status message.

    3. Test case: `delete 0`<br>
       Expected: No patient is edited. Error details are shown in the response box.

>     4. Other incorrect edit commands to try: `delete`, `delete x`, `delete abcd` (where x is larger than the list size, and abcd are any alphabets)<br>

Reviewer: ```suggestion
    4. Other incorrect delete commands to try: `delete`, `delete x`, `delete abcd` (where x is larger than the list size, and abcd are any alphabets)<br>
```

        Expected: No doctor is edited. Error message is shown in the response box: "This doctor already exists in PlannerMD"
       
    7. Test case: `add n/John Doe hp/98765432 eml/johnd@example.com a/311, Kent Ridge Ave 2, #02-25 dob/25/12/0000 t/experienced`
        Expected: No doctor is edited. Error message is shown in the response box: "This doctor already exists in PlannerMD"

    8. Other incorrect edit commands to try: `add`, `add n/Bob hp/`, `add n/` (not all fields, except tags, are filled)<br>
        Expected: Similar to previous.
       
### Deleting a doctor  <a name="delete-doctor-manual-testing"/>
1. Deleting a doctor while all doctors are being shown
    1. Prerequisites: `toggle` to the `Doctors` tab. List all doctors using the `list` command. There must be multiple doctors. 

    2. Test case: `delete 1`<br>
       Expected: First doctor is deleted from the doctors list. Details of the deleted doctor are shown in the status message.

    3. Test case: `delete 0`<br>
       Expected: No doctor is edited. Error details are shown in the response box.

>     4. Other incorrect edit commands to try: `delete`, `delete x`, `delete abcd` (where x is larger than the list size, and abcd are any alphabets)<br>

Reviewer: ```suggestion
    4. Other incorrect delete commands to try: `delete`, `delete x`, `delete abcd` (where x is larger than the list size, and abcd are any alphabets)<br>
```

        Expected: Patient named Bobby Doe is added successfully. Details of the added patient are shown in the status message.

    4. Test case: `add n/John Doe hp/999 eml/johnd@example.com a/311, Clementi Ave 2, #02-25 dob/20/07/1964 t/vaccinated risk/LOW`
       Expected: Patient named John Doe is added successfully. Details of the added patient are shown in the status message.

    5. Test case: `add n/John Doe hp/98765432 eml/johndoeeeee@example.com a/311, Clementi Ave 2, #02-25 dob/20/07/1964 t/vaccinated risk/LOW`
       Expected: Patient named John Doe is added successfully. Details of the added patient are shown in the status message.

    6. Test case: `add n/John Doe hp/98765432 eml/johnd@example.com a/311, Clementi Ave 2, #02-25 dob/20/07/1964 t/vaccinated risk/LOW`
       Expected: No Patient is added. Error message is shown in the response box: "This patient already exists in PlannerMD"

    7. Test case: `add n/John Doe hp/98765432 eml/johnd@example.com a/311, Kent Ridge Ave 2, #02-25 dob/25/12/0000 t/vaccinated risk/LOW`
       Expected: No Patient is added. Error message is shown in the response box: "This patient already exists in PlannerMD"

>     8. Other incorrect edit commands to try: `add`, `add n/Bob hp/`, `add n/` (not all fields, except tags and risk, are filled)<br>
>        Expected: Similar to previous.

Reviewer: ```suggestion
    8. Other incorrect add commands to try: `add`, `add n/Bob hp/`, `add n/` (not all fields, except tags and risk, are filled)<br>
       Expected: No Patient is added. Error message is shown in the response box: "Invalid command format!..."
```

>        Expected: Similar to previous.

Reviewer: There are still some "edit" to be changed haha

      Expected: Doctor named Joe Doe is added successfully. Details of the added doctor are shown in the status message.
       
    4. Test case: `add n/John Doe hp/999 eml/johnd@example.com a/311, Clementi Ave 2, #02-25 dob/20/07/1964 t/experienced`
          Expected: Doctor named John Doe is added successfully. Details of the added doctor are shown in the status message.

    5. Test case: `add n/John Doe hp/98765432 eml/johndoeeeee@example.com a/311, Clementi Ave 2, #02-25 dob/20/07/1964 t/experienced`
        Expected: Doctor named John Doe is added successfully. Details of the added doctor are shown in the status message.
       
    6. Test case: `add n/John Doe hp/98765432 eml/johnd@example.com a/311, Clementi Ave 2, #02-25 dob/20/07/1964 t/experienced`
        Expected: No doctor is added. Error message is shown in the response box: "This doctor already exists in PlannerMD"
       
    7. Test case: `add n/John Doe hp/98765432 eml/johnd@example.com a/311, Kent Ridge Ave 2, #02-25 dob/25/12/0000 t/experienced`
        Expected: No doctor is added. Error message is shown in the response box: "This doctor already exists in PlannerMD"

>     8. Other incorrect edit commands to try: `add`, `add n/Bob hp/`, `add n/` (not all fields, except tags, are filled)<br>

Reviewer: Here also
